components/match2/wikis/rocketleague/match_group_input_custom.lua

```diff
@@ -207,6 +213,17 @@ function matchFunctions.getExtraData(match)
 	return match
 end
 
+function getPlayerFlag(name)
```

Comment:
Broadcaster table and the matches can be on different pages. Probably better to query LPDB broadcaster table

Reply:
i was thinking of doing
if entry
elseif var
elseif broadcastertable query --> set var
elseif query from lpdb_player --> set var
else blank

Reply:
No player lpdb query imo. All caster's should be in the broadcaster table (which uses Caster pages and other tournaments to gather information).

Reply:
okay so basically something like this:
```lua
function p._getCasterFlag(name, flag)
	if String.isNotEmpty(flag) then
		Variables.varDefine(name .. '_flag', flag)
		return flag
	end
	flag = Variables.varDefault(name .. '_flag')
	if String.isEmpty(flag) then
		local pageName = mw.ext.TeamLiquidIntegration.resolve_redirect(name):gsub(' ', '_')
		local data = mw.ext.LiquipediaDB.lpdb('broadcasters', {
			conditions = '[[page::' .. pageName .. ']]',
			query = 'flag',
			limit = 1
		})
		if type(data) == 'table' and data[1] then
			flag = (data[1] or {}).flag
			if String.isNotEmpty(flag) then
				Variables.varDefine(name .. '_flag', flag)
			end
		end
	end

	return flag
end
```

it would reduce the need for such queries to a max of 1 per caster per page instead of 1 per caster per match they casted
(the flag being passed to the function would just be the manual input one, so the call line for this function would look like:
`flag = p._getCasterFlag(name, match[key .. 'flag'])`)

Reply:
The query should be limited to this tournament as well (using [[parent::]]).
Display Name should probably be automatically retrieved from the same as well.